src/components/Footer/constants.ts

```diff
@@ -22,13 +21,13 @@ type SocialLink = {
 
 export const FOOTER_LINKS: { home: Link[]; app: Link[] } = {
   home: [
-    { text: defineMessage({ message: "Terms and Conditions" }), link: "/terms-and-conditions" },
-    { text: defineMessage({ message: "Referral Terms" }), link: "/referral-terms" },
-    { text: defineMessage({ message: "Media Kit" }), link: "https://gmxio.gitbook.io/gmx/media-kit", external: true },
+    { text: t`Terms and Conditions`, link: "/terms-and-conditions" },
```

Comment:
but here FOOTER_LINKS should be inside getFooterLinks function?
Otherwise they are only initialized once?

Reply:
Moved it. Thanks for noticing.